articles/153_roslaunch_remote_launch.md

```diff
@@ -0,0 +1,94 @@
+# Multi-Machine and Remote Launching
+This article describes the design of the components of the ros2 launch system related to the remote execution of processes.
+
+## Goals
+ROS 1's launch system provided the means to launch nodes remotely via SSH by defining a "machine" and referencing that machine in the declaration of each node meant to run on it.
+The machine declaration would contain the information necessary to execute processes remotely, such as address, username, password, etc.
+The goal of this design is to enable the same ability to launch nodes on remote machines, as well as to provide infrastructure for more advanced capabilities.
+
+- Monitor the status and manage lifecycles of nodes across systems
+- Shutdown nodes remotely
+- Ability to detach the machine used to launch a system and have the system continue running
+    - Be able to reconnect to the system to monitor and manage nodes running remotely
+
+
+To achieve our goals, this update will consist of two primary components: an ExecuteRemote action, and a LaunchServiceNode.
+
+The ExecuteRemote action will provide the basic capaiblity to start processes on remote machines.
+It will build off of the refactor that is described [here](https://github.com/ros2/design/pull/272) and include a machine object as an additional parameter providing implementation-specific connection information.
+Since we don't want to limit remote execution to a single protocol, ExecuteRemote will be a base class and we will create an implementation using SSH which we will call ExecuteRemoteSSH.
+The machine class will also be a base class for protocol-specific implementations.
+Together these components will provide the means to execute processes remotely and should cover most basic use cases.
+
+The [LaunchServiceNode(s)](154_roslaunch_remote_launch_service_node.md) will expose ROS2 services and topics for handling startup and shutdown of processes on remote machines.
```

Comment:
It may be easier to evaluate this design tie-in if the linked document was available?